From: "Barak A. Pearlmutter" <barak+git@pearlmutter.net>
Date: Wed, 16 Sep 2015 10:05:30 +0100
Subject: autoload set-key

Add a global-set-key C-= and mark it for inclusion in the autoloads
file.  This will set the keybinding when the package is installed,
so users won't have to monkey with their init files.
---
 expand-region.el | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/expand-region.el b/expand-region.el
index 1ff1d9f..6b5b71b 100644
--- a/expand-region.el
+++ b/expand-region.el
@@ -140,6 +140,9 @@
 (require 'er-basic-expansions)
 
 ;;;###autoload
+(global-set-key (kbd "C-=") 'er/expand-region)
+
+;;;###autoload
 (defun er/expand-region (arg)
   "Increase selected region by semantic units.
 
